doc/api/child_process.md

```diff
@@ -110,6 +111,13 @@ exec('my.bat', (err, stdout, stderr) => {
   }
   console.log(stdout);
 });
+
+// Script with spaces in the filename:
+const bat = spawn('"my script.cmd"', ['a', 'b'], { shell:true });
+// or:
+exec('"my script.cmd" a b', (err, stdout, stderr) => {
```

Comment:
Maybe just for my own understading… these two variants won’t be distinguishable, right?